autoPyTorch/pipeline/components/setup/network_backbone/ResNetBackbone.py

```diff
@@ -137,22 +152,28 @@ def get_hyperparameter_search_space(
         # The number of groups that will compose the resnet. That is,
         # a group can have N Resblock. The M number of this N resblock
         # repetitions is num_groups
-        min_num_gropus, max_num_groups = num_groups.value_range
+        _, max_num_groups = num_groups.value_range
         num_groups = get_hyperparameter(num_groups, UniformIntegerHyperparameter)
 
         add_hyperparameter(cs, activation, CategoricalHyperparameter)
         cs.add_hyperparameters([num_groups])
 
+        # activation controlled batch normalization
+        add_hyperparameter(cs, use_batch_norm, CategoricalHyperparameter)
+
         # We can have dropout in the network for
         # better generalization
         use_dropout = get_hyperparameter(use_dropout, CategoricalHyperparameter)
         cs.add_hyperparameters([use_dropout])
 
-        use_shake_shake = get_hyperparameter(use_shake_shake, CategoricalHyperparameter)
-        use_shake_drop = get_hyperparameter(use_shake_drop, CategoricalHyperparameter)
+        use_sc = get_hyperparameter(use_skip_connection, CategoricalHyperparameter)
+        mb_choice = get_hyperparameter(multi_branch_choice, CategoricalHyperparameter)
         shake_drop_prob = get_hyperparameter(max_shake_drop_probability, UniformFloatHyperparameter)
-        cs.add_hyperparameters([use_shake_shake, use_shake_drop, shake_drop_prob])
-        cs.add_condition(CS.EqualsCondition(shake_drop_prob, use_shake_drop, True))
+        cs.add_hyperparameters([use_sc, mb_choice, shake_drop_prob])
+        cs.add_condition(CS.EqualsCondition(mb_choice, use_sc, True))
+        # TODO check if shake_drop is as an option in mb_choice
```

Comment:
Sorry, what do you mean here?

Reply:
Hmm, I mean when `mb_choice` (multibranch choice, maybe should be renamed) has no `shake-drop` in the value_range, the condition that adds the `shake_drop_prob` in line 176 will fail. This can be fixed with a simple if check, however, at that time, I wanted to do it quite fast. Maybe it makes sense to add that condition now